client/src/lib/constants.js

```diff
@@ -0,0 +1 @@
+export const SPLITMODE_BREAKPOINT = 800;
```

Comment:
It'd be good to add a comment describing what this is for too :)

Reply:
I'd love to see this configurable somehow (maybe stored in the redux store instead), but out of scope for this pull request I think